docs/docs/rest-api/public/api/v2/pods.raml

```diff
@@ -0,0 +1,22 @@
+#%RAML 1.0
+title: v2 pod API
+mediaType: application/json
+types:
+  Pod: !include types/pod.raml
+  PathId: !include types/pathId.raml
+/{id}:
+  uriParameters:
+    id:
+      type: PathId
```

Comment:
so our `PathId` could have `/` chars in it. Is that legal in RAML? I couldn't find a section of the doc that explained if that was allowed